resources/metadata/abapEnvironmentAssemblePackages.yaml

```diff
@@ -0,0 +1,115 @@
+metadata:
+    name: abapEnvironmentAssemblePackages
+    description: Assembly of installation, support package or patch in SAP Cloud Platform ABAP Environment system
+    longDescription: |
+        Assembly of of a list of installations, support packages or patches in SAP Cloud Platform ABAP Environment system      
```

Comment:
Maybe you can further elaborate what the step is doing here in the long description.